sw.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+importScripts('/sw-toolbox.js');
+
+const config = {
+  offlinePage: '/offline/'
+}
+
+const IGNORED_URLS = []
+
+config.filesToCache = [
+    '/',
+    '/offline/',
```

Comment:
As my follow up commit (32c22ac) illustrates, one should verify the exact file paths your live site will have. For example, my new offline page, while working on my local preview at `/offline/`, needed to be switched to `/offline.html` to work correctly on the live site.